docs/TheoryDeepPixels.rst

```diff
@@ -0,0 +1,491 @@
+======================
+Theory of Deep Samples
+======================
+
+
+This document derives the techniques for splitting and combining two non-solid samples of
+equal depth and thickness. These should should be used by deep image "flattening" algorithms to compute the combined colour of two samples. The formulas are defined in the document "Interpreting OpenEXR Deep Pixels". This document derives those formulas, and is for information only.
+
+
+
+Definitions
+===========
+
+-  ratio of length of original sample to a subsample:
+   :math: 0\le r \le 1
+
+-  Colour of sample :math:`i`
+
+-  Colour of a subsample of :math:`i`
+
+-  alpha value of sample :math:`i`
+
+-  alpha value of a subsample of :math:`i`
+
+-  transparency/transmission of sample :math:`i`; :math:`T_a=1-\alpha_a`
+-  colour per unit length (‘instantaneous colour’) of :math:`i`
+-  transparency per unit length (‘intenstaneous transparency’) of
+   :math:`i`
+
+Subscripts :math:`_a` and :math:`_b` refer to the input samples;
+subscript :math:`_c` refers to the combined output sample
+
+Sample model
```

Comment:
This may be an Americanism, but the style used in the other docs is to capitalize all works in the section headers: "Sample Model".